.appveyor.yml

```diff
@@ -0,0 +1,40 @@
+# Build matrix / environment variables are explained on:
```

Comment:
The CMake build configuration is initially based off @pwnall's fork, so that would be ideal.

Reply:
@pwnall can you take a look at the config stuff?  Though maybe later since I think there will be a resubmitted PR after making the changes I suggested.